test/metabase/api/dataset_test.clj

```diff
@@ -48,7 +52,9 @@
        :else
        [k v]))))
 
-(defn- most-recent-query-execution [] (db/select-one QueryExecution {:order-by [[:id :desc]]}))
+(defn- most-recent-query-execution []
+  (Thread/sleep 100)
+  (db/select-one QueryExecution {:order-by [[:id :desc]]}))
```

Comment:
Have you considered ordering by `started_at`? I'm really uneasy about using sleep here as it's a great recipe for intermittent failures.

Reply:
I'll try that :+1:.

Reply:
Looks like that did the trick :tada: